.github/workflows/chrome-store-upload.yaml

```diff
@@ -0,0 +1,29 @@
+name: Upload To Chrome Web Store
+# This workflow is triggered on pushes to the repository.
+on: 
+  push:
+    tags:
+      - v1*
```

Comment:
if I'm reading this correctly, this will only trigger for `v1**` pattern-matched tags, correct? Any reason why not to generalize to `v*`?

Reply:
Not really. A lot of this was back and forth testing of what did and did not work based on examples. Must have just missed it.